ociinstaller/steampipeimage.go

```diff
@@ -141,28 +141,28 @@ func getDBImageData(layers []ocispec.Descriptor) (*DbImage, error) {
 	return res, nil
 }
 
-func getHubImageData(layers []ocispec.Descriptor) (*HubImage, error) {
+func getFdwImageData(layers []ocispec.Descriptor) (*HubImage, error) {
 	res := &HubImage{}
 
 	// get the binary (steampipe-postgres-fdw.so) info
-	foundLayers := findLayersForMediaType(layers, MediaTypeForPlatform("fdw"))
+	foundLayers := findLayersForMediaType(layers, MediaTypeForPlatform("fdw")[0])
 	if len(foundLayers) != 1 {
-		return nil, fmt.Errorf("Invalid Image - Image should contain 1 binary file per platform, found %d", len(foundLayers))
+		return nil, fmt.Errorf("invalid image - Image should contain 1 binary file per platform, found %d", len(foundLayers))
```

Comment:
`invalid image - should contain 1 binary file per platform, found %d`

Reply:
and for others below

Reply:
doen